### DISCUSSION TITLE
migrate errors

### DISCUSSION THREAD

**Discussion by kakarot-dev** [prisma/prisma#8853] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/76399483/130351566-950cc480-2a64-4a95-bf2a-47bde6021523.png)

some errors happening while trying to run `yarn prisma migrate` on a cockroach db cloud cluster

any help please?

or can i do thses things without migrate?

**Comment by kakarot-dev** [comment]
anyone?

**Comment by ryands17** [comment, marked as answer]
@kakarot-dev 👋 

We don't support Cockroach DB yet. It would be great if you could add a 👍 and follow this request for more updates:
https://github.com/prisma/prisma/issues/4702

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
